src/components/Navbar/index.tsx

```diff
@@ -1,16 +1,19 @@
 import React from 'react'
-import Logo from '../Logo'
+import NavbarLogo from '../Logo'
 import NavbarMenu from './Menu'
 import NavbarTool from './Tool'
+import Hamburger from './Hamburger'
 
 const Navbar = () => {
   return (
     <nav className="navbar">
-      <Logo />
-      <NavbarMenu />
-      <NavbarTool />
+      <div className="navbar__container">
+        <Hamburger />
```

Comment:
Since you decided to use the word `Navbar` as a prefix, should this one also be added?
```suggestion
        <NavbarHamburger />
```

Reply:
Agree